introduction.md

```diff
@@ -1,2 +1,5 @@
 I introduce myself.
 Ruby
+
+Deadly Skills
+*python
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?